docs/rootless.md

```diff
@@ -160,6 +160,18 @@ systemctl --user daemon-reload
 systemctl --user restart containerd
 ```
 
+### Rootlesskit Network Design
+
+* `Nerdctl` supports rootlesskit `detach-netns` mode to ensure controlling network access for the container in detach mode. If using the non `detach-netns` mode, all steps mentioned below would be executed within the RootlessKit `Child` network namespace directly.
+
+![rootlessKit-network-design.png](assets/images/rootlessKit-network-design.png)
+
+- Rootlesskit Parent NetNS and Child NetNS are already configured by the startup script [containerd-rootless.sh](https://github.com/fahedouch/nerdctl/blob/3e5b0853067d38b2ab1a456bdc31fd08d0462aae/extras/rootless/containerd-rootless.sh)
+- Rootlesskit Parent NetNS is the host network namespace
+- step3: `runc` creates container with dedicated namespaces (e.g network ns) in the Parent netns
+- step4: `runc` nsenter Rootlesskit Child NetNS before triggering nerdctl ocihook
+- step6: CNI configures container network namespace: create network interfaces `eth0` -> `veth0` -> `nerdctl0`
```

Comment:
Lacks step 1, 2, and 5

Reply:
it was trivial to me but ok if you want!